.github/workflows/docker.yml

```
@@ -5,6 +5,7 @@ on:
       - main
 jobs:
     docker:
+        if: github.repository == 'spaceninjaserver/SpaceNinjaServer'
```

yamllint

[warning] wrong indentation: expected 6 but found 8

(indentation)